include/model/Note.hpp

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include <set>
+#include <string>
+
+#include "model/ModelClock.hpp"
+#include "model/NoteId.hpp"
+#include "model/TagId.hpp"
+#include "model/Timestamp.hpp"
+
+namespace doma {
+namespace model {
+
+class Note {
+ public:
+  Note(const NoteId id, const std::string &name, const std::string &text);
+
+  bool isDone() const;
+  NoteId getId() const;
+  Timestamp getDateCreate() const;
+  Timestamp getDateChange() const;
+  Timestamp getDateDone() const;
+  std::string &getName();
+  std::string &getText();
+  std::set<TagId> &getTagIds();
+
+  void changeName(const std::string &name);
+  void changeText(const std::string &name);
+  bool addTag(const TagId tagId);
```

Comment:
please rename add `try` prefix because `addTag` it sounds like ended action and after executing the method we expect what a tab will be added.

But it may not be so.

So we are _try_-ing to _add_ a _tag_ == `tryAddTag()`

Reply:
done